app/controllers/concerns/vet360/writeable.rb

```diff
@@ -22,6 +22,22 @@ def write_to_vet360_and_render_transaction!(type, params, http_verb: 'post')
       render_new_transaction!(type, response)
     end
 
+    # Temporary method for debugging during UAT
+    #
+    def log_profile_data_to_sentry(response)
+      log_message_to_sentry(
+        'Profile controller bug',
+        :info,
+        {
+          controller: self.class.to_s,
+          response: response,
+          params: params,
+          user: @current_user.inspect
```

Comment:
Although this is "Temporary method for debugging during UAT" logs are forever 😃so it needs a conditional logic not to log in production (which I realize may defeat the purpose). Even if it's a UAT user they're still a real person and it can trigger a painful audit if their PII is logged. As mentioned in slack an alternative is to log just enough info to a database column to aid in debugging (and encrypt things like ssn).

Reply:
Good catch. Can we get this rolled back before deployment tomorrow?